cobra/test/test_solver_model.py

```diff
@@ -498,12 +498,18 @@ def test_add_metabolites_combine_false(self, model):
     #            ).as_coefficients_dict()
 
     def test_remove_from_model(self, model):
-        pgi = model.reactions.PGI
-        pgi.remove_from_model()
-        assert pgi.model is None
-        assert not ("PGI" in model.reactions)
-        assert not (pgi.id in model.variables)
-        assert not (pgi.reverse_id in model.variables)
+        with model:
+            pgi = model.reactions.PGI
+            pgi.remove_from_model()
+            assert pgi.model is None
+            assert not ("PGI" in model.reactions)
+            assert not (pgi.id in model.variables)
+            assert not (pgi.reverse_id in model.variables)
+
+        assert ("PGI" in model.reactions)
+        assert (pgi.id in model.variables)
+        assert (pgi.reverse_id in model.variables)
+        pgi.forward_variable
```

Comment:
`assert pgi.forward_variable is not None` or similar? As written the statement doesn't  look like it has an effect, I know what it's meant for but looks a bit confusing..